dynamic_programming/hamiltons-cycle.cpp

```diff
@@ -1,9 +1,15 @@
 /**
- * @author vakhokoto
+ * @file
+ * @brief The implementation of hamilton's cycle dynamic solution for vertices number less than 20.
+ * @details 
+ * I use 2^n * n matrix it's and for every [i, j] (i < 2^n and j < n) in the matrix I 
+ * store there if it is possible to get to all vertices on which position in i's binary 
```

Comment:
```suggestion
 * we store there if it is possible to get to all vertices on which position in i's binary 
```
I am unable to understand what this sentence means and unable to correct it